packages/core/admin/ee/server/tests/review-workflows.test.api.js

```diff
@@ -373,4 +373,38 @@ describeOnCondition(edition === 'EE')('Review workflows', () => {
       expect(response.length).toEqual(0);
     });
   });
+
+  describe('Creating an entity in a review workflow content type', () => {
+    let response;
+
+    beforeAll(async () => {
+      await updateContentType(productUID, {
+        components: [],
+        contentType: { ...model, reviewWorkflows: true },
+      });
+      await restart();
+
+      await createEntry(productUID, { name: 'Product' });
+      await createEntry(productUID, { name: 'Product 1' });
+      await createEntry(productUID, { name: 'Product 2' });
+    });
+
+    test('when review workflows is enabled on a content type, new entries should be added to the first stage of the default workflow', async () => {
+      response = await requests.admin({
+        method: 'GET',
+        url: `/content-type-builder/content-types/api::product.product`,
+      });
+
+      expect(response.body.data.schema.reviewWorkflows).toBeTruthy();
+
+      response = await getRWMorphTableResults(strapi.db.getConnection());
+
+      expect(response.length).toEqual(6);
+      for (let i = 0; i < response.length; i += 1) {
+        const entry = response[i];
+        expect(entry.related_id).toEqual(i + 1);
+        expect(entry.order).toEqual(1);
+      }
```

Comment:
last comment I promise 🙈 

Why not looking at the response of the admin api? it shoudl contain the `strapi_reviewWorkflow_stage` attribute. 

That way this test doesnt have to know that attribute is using a morph table.